app/controllers/users/posts_controller.rb

```diff
@@ -0,0 +1,39 @@
+class Users::PostsController < ApplicationController
+  before_action :set_post, only: [ :edit, :update, :destroy ]
+
+  def new
+    @post = Post.new
```

Comment:
このコントローラが扱うpostはすべてcurrent_userから引く
```suggestion
    @post = current_user.posts.new
```